Xplat/src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -3036,7 +3036,7 @@
   "botania.tagline.blackHoleTalisman": "Infinite storage of a block in a single slot",
   "botania.page.blackHoleTalisman0": "The void is a massive space full of... nothing. Really. There's nothing there. But with a bit of ingenuity, this nothingness can be exploited to store blocks for you to your heart's content.$(p)The $(item)Black Hole Talisman$(0) utilizes powerful Gaia and Ender magics to store a virtually infinite quantity of a single type of block.",
   "botania.page.blackHoleTalisman1": "Right-clicking a block with an empty Talisman will set it to store that type of block, and sneak-right clicking the Talisman in the air will enable or disable it. When enabled, a Talisman will absorb any blocks of its given type from its user's inventory.$(p)Placing the item in a crafting grid will yield its stored blocks, a stack at a time.",
-  "botania.page.blackHoleTalisman2": "Any blocks stored in a Talisman can be placed by simply right-clicking with it.$(p)Right-clicking the Talisman on an inventory will dump up to a stack of blocks into that inventory. Finally, a Talisman can provide a $(l:tools/exchange_rod)$(item)Rod of the Shifting Crust$(0)$(/l) with the blocks it contains.",
+  "botania.page.blackHoleTalisman2": "Any blocks stored in a Talisman can be placed by simply right-clicking with it.$(p)Sneak-right clicking the Talisman on an inventory will dump up to a stack of blocks into that inventory. Finally, a Talisman can provide a $(l:tools/exchange_rod)$(item)Rod of the Shifting Crust$(0)$(/l) with the blocks it contains.",
```

Comment:
Should be `Sneak right-clicking`, not `Sneak-right clicking`

Reply:
![image](https://user-images.githubusercontent.com/43409914/188239598-15c17691-df36-484a-9de3-d0935a649706.png)

Then why are there 33 entries that use `Sneak-right clicking` and 0 entries that use `Sneak right-clicking`.

Reply:
ok, this is fine for now then, though I think that's the wrong version to use.

Reply:
Yeah true, but I didn't want to go against the standard that was already established.